[2024-06-20T09:54:08.514+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-20T09:54:08.663+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.raw_orders.test manual__2024-06-20T09:52:24.201945+00:00 [queued]>
[2024-06-20T09:54:08.717+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ELT-PROJECT.transform.raw_orders.test manual__2024-06-20T09:52:24.201945+00:00 [queued]>
[2024-06-20T09:54:08.719+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-20T09:54:08.797+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestLocalOperator): transform.raw_orders.test> on 2024-06-20 09:52:24.201945+00:00
[2024-06-20T09:54:08.832+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=8736) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-20T09:54:08.852+0000] {standard_task_runner.py:63} INFO - Started process 8753 to run task
[2024-06-20T09:54:08.849+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'ELT-PROJECT', 'transform.raw_orders.test', 'manual__2024-06-20T09:52:24.201945+00:00', '--job-id', '542', '--raw', '--subdir', 'DAGS_FOLDER/elt.py', '--cfg-path', '/tmp/tmpah9dge6k']
[2024-06-20T09:54:08.862+0000] {standard_task_runner.py:91} INFO - Job 542: Subtask transform.raw_orders.test
[2024-06-20T09:54:08.940+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-06-20T09:54:09.129+0000] {task_command.py:426} INFO - Running <TaskInstance: ELT-PROJECT.transform.raw_orders.test manual__2024-06-20T09:52:24.201945+00:00 [running]> on host airflow-scheduler
[2024-06-20T09:54:09.690+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Data-Ninja' AIRFLOW_CTX_DAG_ID='ELT-PROJECT' AIRFLOW_CTX_TASK_ID='transform.raw_orders.test' AIRFLOW_CTX_EXECUTION_DATE='2024-06-20T09:52:24.201945+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-20T09:52:24.201945+00:00'
[2024-06-20T09:54:09.693+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-20T09:54:09.713+0000] {local.py:174} INFO - Could not import dbtRunner. Falling back to subprocess for invoking dbt.
[2024-06-20T09:54:09.717+0000] {local.py:292} INFO - Cloning project to writable temp directory /tmp/tmpzmi52ut_ from /opt/airflow/dags/dbt/dbt_project
[2024-06-20T09:54:09.730+0000] {local.py:303} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/ELT-PROJECT__transform/target/partial_parse.msgpack
[2024-06-20T09:54:09.855+0000] {base.py:84} INFO - Using connection ID 'snow_conn' for task execution.
[2024-06-20T09:54:09.859+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-06-20T09:54:09.862+0000] {base.py:225} INFO - Using real values for profile dbt_project
[2024-06-20T09:54:09.880+0000] {config.py:291} INFO - Creating temporary profiles.yml with use_mock_values=False at /tmp/tmp025suow8/profiles.yml with the following contents:
dbt_project:
    outputs:
        dev:
            account: sk84886.europe-west2.gcp
            database: ELT_PROJECT
            password: '{{ env_var(''COSMOS_CONN_SNOWFLAKE_PASSWORD'') }}'
            role: SYSADMIN
            schema: PUBLIC
            type: snowflake
            user: DATANINJA
            warehouse: TRANSFORM
    target: dev

[2024-06-20T09:54:09.900+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpzmi52ut_', '--profiles-dir', '/tmp/tmp025suow8', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpzmi52ut_
[2024-06-20T09:54:09.902+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-06-20T09:54:09.905+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'deps', '--project-dir', '/tmp/tmpzmi52ut_', '--profiles-dir', '/tmp/tmp025suow8', '--profile', 'dbt_project', '--target', 'dev']
[2024-06-20T09:54:09.961+0000] {subprocess.py:84} INFO - Command output:
[2024-06-20T09:54:18.201+0000] {subprocess.py:94} INFO - [0m09:54:18  Running with dbt=1.8.2
[2024-06-20T09:54:18.713+0000] {subprocess.py:94} INFO - [0m09:54:18  Updating lock file in file path: /tmp/tmpzmi52ut_/package-lock.yml
[2024-06-20T09:54:18.738+0000] {subprocess.py:94} INFO - [0m09:54:18  Installing dbt-labs/dbt_utils
[2024-06-20T09:54:19.714+0000] {subprocess.py:94} INFO - [0m09:54:19  Installed from version 0.8.6
[2024-06-20T09:54:19.720+0000] {subprocess.py:94} INFO - [0m09:54:19  Updated version available: 1.2.0
[2024-06-20T09:54:19.724+0000] {subprocess.py:94} INFO - [0m09:54:19
[2024-06-20T09:54:19.740+0000] {subprocess.py:94} INFO - [0m09:54:19  Updates available for packages: ['dbt-labs/dbt_utils']
[2024-06-20T09:54:19.742+0000] {subprocess.py:94} INFO - Update your versions in packages.yml, then run dbt deps
[2024-06-20T09:54:22.341+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-06-20T09:54:22.346+0000] {local.py:254} INFO - Update your versions in packages.yml, then run dbt deps
[2024-06-20T09:54:22.353+0000] {local.py:247} INFO - Trying to run the command:
 ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'raw_orders', '--project-dir', '/tmp/tmpzmi52ut_', '--profiles-dir', '/tmp/tmp025suow8', '--profile', 'dbt_project', '--target', 'dev']
From /tmp/tmpzmi52ut_
[2024-06-20T09:54:22.364+0000] {subprocess.py:60} INFO - Tmp dir root location: 
 /tmp
[2024-06-20T09:54:22.381+0000] {subprocess.py:73} INFO - Running command: ['/opt/airflow/dbt_venv/bin/dbt', 'test', '--models', 'raw_orders', '--project-dir', '/tmp/tmpzmi52ut_', '--profiles-dir', '/tmp/tmp025suow8', '--profile', 'dbt_project', '--target', 'dev']
[2024-06-20T09:54:22.483+0000] {subprocess.py:84} INFO - Command output:
[2024-06-20T09:54:33.600+0000] {subprocess.py:94} INFO - [0m09:54:33  Running with dbt=1.8.2
[2024-06-20T09:54:35.688+0000] {subprocess.py:94} INFO - [0m09:54:35  Registered adapter: snowflake=1.8.3
[2024-06-20T09:54:39.453+0000] {subprocess.py:94} INFO - [0m09:54:39  Found 20 models, 27 data tests, 11 sources, 682 macros
[2024-06-20T09:54:39.469+0000] {subprocess.py:94} INFO - [0m09:54:39
[2024-06-20T09:54:46.379+0000] {subprocess.py:94} INFO - [0m09:54:46  Concurrency: 1 threads (target='dev')
[2024-06-20T09:54:46.382+0000] {subprocess.py:94} INFO - [0m09:54:46
[2024-06-20T09:54:46.412+0000] {subprocess.py:94} INFO - [0m09:54:46  1 of 1 START test not_null_raw_orders_ORDERID .................................. [RUN]
[2024-06-20T09:54:48.474+0000] {subprocess.py:94} INFO - [0m09:54:48  1 of 1 PASS not_null_raw_orders_ORDERID ........................................ [[32mPASS[0m in 2.06s]
[2024-06-20T09:54:48.490+0000] {subprocess.py:94} INFO - [0m09:54:48
[2024-06-20T09:54:48.494+0000] {subprocess.py:94} INFO - [0m09:54:48  Finished running 1 test in 0 hours 0 minutes and 9.02 seconds (9.02s).
[2024-06-20T09:54:48.859+0000] {subprocess.py:94} INFO - [0m09:54:48
[2024-06-20T09:54:48.861+0000] {subprocess.py:94} INFO - [0m09:54:48  [32mCompleted successfully[0m
[2024-06-20T09:54:48.866+0000] {subprocess.py:94} INFO - [0m09:54:48
[2024-06-20T09:54:48.870+0000] {subprocess.py:94} INFO - [0m09:54:48  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-06-20T09:54:50.672+0000] {subprocess.py:98} INFO - Command exited with return code 0
[2024-06-20T09:54:50.674+0000] {local.py:254} INFO - [0m09:54:48  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2024-06-20T09:54:50.718+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2024-06-20T09:54:50.719+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2024-06-20T09:54:50.736+0000] {local.py:349} INFO - Inlets: [Dataset(uri='snowflake://sk84886.europe-west2.gcp/ELT_PROJECT.raw.raw_orders', extra=None)]
[2024-06-20T09:54:50.738+0000] {local.py:350} INFO - Outlets: []
[2024-06-20T09:54:50.740+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-06-20T09:54:51.181+0000] {dag.py:3954} INFO - Setting next_dagrun for ELT-PROJECT to None, run_after=None
[2024-06-20T09:54:51.264+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-20T09:54:51.326+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ELT-PROJECT, task_id=transform.raw_orders.test, run_id=manual__2024-06-20T09:52:24.201945+00:00, execution_date=20240620T095224, start_date=20240620T095408, end_date=20240620T095451
[2024-06-20T09:54:51.381+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-20T09:54:51.475+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-20T09:54:51.480+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
